_releases/3.12.0.md

```diff
@@ -0,0 +1,100 @@
+
+- User cannot delete his account in case a Contract does not have a user with the contract-owner role. Now the user with the *Contract Owner* role cannot perform a permanent deletion his/her account in case there is no other *Contract Owner* left in a particular Contract. For the other contract roles delete users account function available without any checks.
+
+- Wording: replace admin with owner in API response. Replaced an *admin* with an *owner* in an API response to (Delete a user)[https://api.elastic.io/docs/v2/#delete-a-user] endpoint in case a deleted user is the last Owner in a Contract.
+
+- The easiest way to kill elastic.io webhooks. `404 Not Found` error returns in case an invalid webhook-url is being send to the platform.
+
+
+## New Features <img width="25" alt ="features" src="https://user-images.githubusercontent.com/28867440/42755172-d8046d70-88ff-11e8-917c-ed1fcf2aac8c.png">
+
+- As the Contract Admin, I want to invite users to contract and workspace in one click right on the Contract Page.
+Now, the Contract Admin is allowed to invite other users to a workspace and a contract simultaneously on the Contract Page.
+
+- Remove the `hideUserSupport` parameter in the Tenant's configuration.
+The `hideUserSupport` parameter is redundant. Therefore, it was expelled from the "Tenant-model". 
+Now, there is the `intercomAppId` parameter that should be used instead. 
+
+- Do not add the intercom script on the page, if the `tenant.hideUserSupport` parameter is set.
+Now, the `intercomAppId` parameter affects an overall appearance of the Intercom's System.
+
+- Invite to a Contract and a Workspace in one click.
+Now, the Contract Admin is allowed inviting any other user to a workspace and a contract simultaneously right on the Workspace Page.
+
+- Extend driver events to have a purpose message from a Pod's container status.
+Common issues related to running steps inside a flow are reflected in the `runlog`. For example, the `Out of Memory` issue will be addressed as *"Component run out of memory and terminated."* This approach can help with improving an overall debugging procedure.
+
+- Check roles usage in the `/v2/tenant/:id/roles`.
+An additional verifications were added for changing roles procedure in the Tenant: 
+    - It is restricted to remove a role from the "Tenant-policy" if any user/contract utilizes it.
+    - In case the role has to be removed from the "Tenant-policy", but one of the `contract.availableRoles` is still using it, the `409` error code will be returned. 
+    
+- Remove `InviteToken`.
+Code refactoring/improving. The `ContractInviteToken` was used as a primary mechanism for invites. However, it was a legacy compatibility code (`src/commons/lib/models/contract.js`) that loads the `InviteToken`. The following means that the `InviteToken` had to be deleted, to make it possible to remove all the code that uses it.
+
+- `Execstats` collection is too large.
```

Comment:
To many internals. I propose to remove this issue completely or rephrase it in such a way, so that it would explain what benefits it introduces.  This would sound like "Improved platform stability" or "Reduced size of one of the database collections" or something like that.